src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/leaked-passwords/leakedPasswordsData.tsx

```diff
@@ -22,32 +22,34 @@ export type LeakedPasswordsContent = {
 export type LeakedPasswordsTypes = "password" | "security-question";
 
 export type LeakedPassword = {
-  type: LeakedPasswordsTypes;
-  title: string;
-  illustration: string;
-  content: LeakedPasswordsContent;
-};
-
-function getLeakedPasswords({
-  dataType,
-  breaches,
-  l10n,
-}: {
   dataType: string;
   breaches: GuidedExperienceBreaches;
   l10n: ExtendedReactLocalization;
-}) {
+  emailAffected: string;
+};
+
+function getLeakedPasswords(props: LeakedPassword) {
+  const { dataType, breaches, l10n, emailAffected } = props;
+
   const findFirstUnresolvedBreach = (breachClassType: LeakedPasswordsTypes) => {
     const leakedPasswordType =
       breachClassType === "password" ? "passwords" : "securityQuestions";
+    const resolvedDataClassName =
+      breachClassType === "password"
+        ? "passwords"
+        : "security-questions-and-answers";
```

Comment:
Why is it `"securityQuestions"` above (L36) but then hyphenated class type of `"security-questions-and-answers"` here?